client/src/components/MultiJurisdictionAudit/Setup/Review/index.tsx

```diff
@@ -128,6 +134,14 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
               <td>Election Name:</td>
               <td>{electionName}</td>
             </tr>
+            <tr>
+              <td>Audit Type:</td>
+              <td>
+                {auditType === 'BALLOT_POLLING'
+                  ? 'Ballot Polling'
+                  : 'Batch Comparison'}
```

Comment:
I noticed that here you have "Ballot Polling" while above you have "Ballot polling" - standardize?

Reply:
@jonahkagan They're on different views, and I just went with what was on the mockup. But it could be both capitalized on the first view too.